mitiq/zne/inference.py

```diff
@@ -920,6 +922,136 @@ def reduce(self) -> float:
         return self._zne_limit
 
 
+class RungeFactory(BatchedFactory):
+    """Factory object implementing another version of Richardson extrapolation.
+    In this version the original set of nodes (scale_factors) are mapped to set
+    of fake nodes known as Chebyshev-Lobatto points. This method gives a better
+    interpolation when there are a lot of nodes, especially close to the
+    edges of the domain.
+
+    Reference:
+        https://www.sciencedirect.com/science/article/abs/pii/S0377042719303449
```

Comment:
There is a specific formatting that we use for including references in function docs, e.g.:
https://github.com/unitaryfund/mitiq/blob/master/mitiq/pec/pec.py

It would be good if you could use a similar style here.